app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala

```diff
@@ -96,4 +105,52 @@ class CompaniesHouseStubController extends InjectedController {
       }
     }
   }
+
+  private def validCompanyProfile(companyNumber: String, companyNameSuffix: Int, address_line_1: String): CompanyProfile = {
+    val registeredOfficeAddressReceivedByIncorporationInformation = RegisteredOfficeAddress(
+      address_line_1 = "10 HIGH STREET",
+      address_line_2 = "address line 2",
+      country = "United Kingdom",
+      locality = "TELFORD",
```

Comment:
Please could we add Telford as address_line_2 and then Shropshire as locality, just incase they need all fields

Reply:
I was going to write to you about all the details .... also premises ... do we need??
<img width="994" alt="Screenshot 2022-05-27 at 13 21 57" src="https://user-images.githubusercontent.com/4958044/170700411-0336d446-a963-42a0-9119-32b6d4f1b7f9.png">

Reply:
okay